### ISSUE TITLE
Dependency Dashboard

### ISSUE THREAD

**Issue by ChristianMurphy** [uPortal-Project/uPortal-start#509]
This issue contains a list of Renovate updates and their statuses.

## Pending Approval

These branches will be created by Renovate only once you click their checkbox below.

 - [ ] chore(deps): update adoptopenjdk docker tag to v16
 - [ ] chore(deps): update casserverversion to v4 (major) (`org.jasig.cas:cas-server-core`, `org.jasig.cas:cas-server-extension-clearpass`)
 - [ ] chore(deps): update dependency com.bmuschko:gradle-docker-plugin to v6
 - [ ] chore(deps): update dependency javax.servlet:javax.servlet-api to v4
 - [ ] chore(deps): update dependency less to v4
 - [ ] chore(deps): update dependency org.apache.portals.pluto:pluto-testsuite to v3
 - [ ] chore(deps): update dependency org.apache.tomcat:tomcat to v10
 - [ ] chore(deps): update dependency org.apereo.service.persondir:person-directory-api to v2
 - [ ] chore(deps): update dependency org.jasig.cas:cas-server-core to v4
 - [ ] chore(deps): update dependency org.jasig.cas:cas-server-extension-clearpass to v4
 - [ ] chore(deps): update dependency org.jasig.cas:cas-server-webapp to v4
 - [ ] chore(deps): update dependency org.springframework:spring-jdbc to v5
 - [ ] chore(deps): update dependency org.webjars.npm:vue to v3
 - [ ] chore(deps): update plutoversion to v3 (major) (`org.apache.portals.pluto:pluto-container-driver-api`, `org.apache.portals.pluto:pluto-taglib`, `org.apache.portals.pluto:pluto-container-api`, `org.apache.portals.pluto:pluto-util`)

## Edited/Blocked

These updates have been manually edited so Renovate will no longer make changes. To discard all commits and start over, check the box below.

 - [ ] chore(deps): update dependency org.apereo.service.persondir:person-directory-api to v1.8.16
 - [ ] chore(deps): update dependency com.bmuschko:gradle-docker-plugin to v3.6.2
 - [ ] chore(deps): update dependency javax.servlet:javax.servlet-api to v3.1.0

## Other Branches

These updates are pending. To force PRs open, check the box below.

 - [ ] chore(deps): update dependency org.apache.tomcat:tomcat to v8.5.69
 - [ ] chore(deps): update dependency org.webjars.npm:vue to v2.6.14
 - [ ] chore(deps): update dependency org.jasig.portlet.notification:notification-portlet-webapp to v4.7.3

## Open

These updates have all been created already. Click a checkbox below to force a retry/rebase of any.

 - [ ] [chore(deps): update dependency org.slf4j:slf4j-api to v1.7.32](../pull/521)
 - [ ] [chore(deps): update dependency org.slf4j:slf4j-api to v1.7.32](../pull/522)
 - [ ] [chore(deps): update dependency org.hsqldb:hsqldb to v2.6.0](../pull/512)
 - [ ] [chore(deps): update dependency org.hsqldb:sqltool to v2.6.0](../pull/513)
 - [ ] [chore(deps): update hsqldbversion to v2.6.0](../pull/515) (`org.hsqldb:hsqldb`, `org.hsqldb:sqltool`)
 - [ ] [chore(deps): update dependency gradle to v7](../pull/516)
 - [ ] **Check this option to rebase all the above open PRs at once**

## Ignored or Blocked

These are blocked by an existing closed PR and will not be recreated unless you click a checkbox below.

 - [ ] [chore(deps): update dependency org.apache.portals.pluto:pluto-container-api to v3](../pull/288)
 - [ ] [chore(deps): update dependency org.apache.portals.pluto:pluto-container-driver-api to v3](../pull/289)
 - [ ] [chore(deps): update dependency org.apache.portals.pluto:pluto-taglib to v3](../pull/290)

**Comment by ChristianMurphy** [comment]
closing in favor of https://github.com/uPortal-Project/uPortal-start/issues/440
